tradesman/model.py

```diff
@@ -46,12 +46,12 @@ def create(self):
         self.build_population_synthesizer_data()
         self.synthesize_population()
 
-    def add_country_borders(self, overwrite=False):
+    def add_country_borders(self, overwrite=False, source="gadm"):
         """Retrieves country borders from www.geoboundarries.org and adds to the model.
```

Comment:
The documentation here is wrong, as we are not retrieving from geoboundaries (there is also an extra r there). 
The "source" argument needs to be in the docstring, along with the options we have for it (gadm/geoboundaries)